content/en/docs/concepts/policy/limit-range.md

```diff
@@ -31,17 +31,78 @@ LimitRange object in that namespace.
 The name of a LimitRange object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
-### Overview of Limit Range
+### Overview of LimitRange
 
 - The administrator creates one LimitRange in one namespace.
```

Comment:
Is this always correct? Can you have more than one?

If we're updating, let's consider this detail in-scope.

Reply:
fixed